qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -1080,50 +1080,65 @@ def GaussianSquareEcho(
     name: Optional[str] = None,
     limit_amplitude: Optional[bool] = None,
 ) -> SymbolicPulse:
-    """An echoed Gaussian square pulse with an active tone overlaid on it.
-    
-    Exactly one of the ``risefall_sigma_ratio`` and ``width`` parameters has to be specified.
-    If ``risefall_sigma_ratio`` is not ``None`` and ``width`` is ``None``:
-    .. math::
-        \\text{risefall} &= \\text{risefall_sigma_ratio} \\times \\text{sigma}\\\\
-        \\text{width} &= \\text{duration} - 2 \\times \\text{risefall}
-    If ``width`` is not None and ``risefall_sigma_ratio`` is None:
-    .. math:: \\text{risefall} = \\frac{\\text{duration} - \\text{width}}{2}
+    """An echoed Gaussian square pulse with an active tone overlaid on it. The ``amp`` and
+    ``angle`` parameters are used to define the amplitude and phase of the echoed Gaussian 
+    square pulse while the ``active_amp`` and ``active_angle`` parameters are used to 
+    define the active tone.
```

Comment:
Generally, it is best to keep the first section short (like one sentence) because it gets pulled out as the short description in the documentation. I would suggest a blank line and after the first sentence, but you give a similar description of the parameters just below so I think we can drop this second sentence.

```suggestion
    """An echoed Gaussian square pulse with an active tone overlaid on it.
```